src/abi/anchor.rs

```diff
@@ -157,7 +157,14 @@ fn idl_instructions(
                 vec![IdlAccountItem::IdlAccount(IdlAccount {
                     name: "dataAccount".to_string(),
                     is_mut: true,
-                    is_signer: func.has_payer_annotation(),
+                    /// With @payer annotation, the account is created on-chain and needs a signer. The client
+                    /// provides an address which does not exist yet, so SystemProgram.CreateAccount is called
+                    /// on-chain.
+                    ///
+                    /// However, if a @seed is provided, then the program can sign for the account
+                    /// using the seed using program derived address (pda) when SystemProgram.CreateAccount is called,
```

Comment:
These two `using` here seem a bit awkward. Is that what you wanted to say?

Reply:
I agree, improved the wording